sdcm/wait.py

```diff
@@ -38,6 +38,7 @@ def wait_for(func, step=1, text=None, timeout=None, throw_exc=True, **kwargs):
     :param timeout: Timeout in seconds
     :param throw_exc: Raise exception if timeout expired, but disrupt_func result is not True
     :param kwargs: Keyword arguments to disrupt_func
+    :param stop_event: instance of threading.Event class to stop retrying
```

Comment:
Here the original logic for parameters is violated.
Would be better to define the `stop_event` on the line 31 explicitly.

Reply:
added